test/maybe.test.js

```diff
@@ -222,4 +222,20 @@ describe('Maybe usage', function() {
       }));
     });
   });
+
+  describe('#toMaybe', function() {
+
+    it('returns Nothing for null', function() {
+      assert.equal(Maybe.toMaybe(null).isNothing, true);
+    });
+    it('returns Nothing for undefined', function() {
+      assert.equal(Maybe.toMaybe(undefined).isNothing, true);
+    });
+    it('returns Just(x) for an arbitrary value x', function() {
+      var just1 = Maybe.toMaybe(1);
+      assert.equal(just1.isJust, true);
+      assert.equal(just1.getOrElse(42), 1);
+    });
+  });
+
```

Comment:
If we simplify the implementation as I suggested in my previous comment, it would be sufficient to assert that `Maybe.toMaybe === Maybe`. :)